### ISSUE TITLE
pyOpenSSL 20 incompatibility

### ISSUE THREAD

**Issue by qsuscs** [plinss/acmebot#47]
Hi,

the latest release explicitly blocks pyOpenSSL >= 20 due to incompatibilities.  After a glance at its changelog and acmebot’s usage of its API, I fail to see the incompatibilities.
Could you please point out said incompatibilities, and if there are none remove this constraint, or else consider this a bug to update the API usage?

Thanks.

**Comment by hyacin75** [comment]
Yeah I've got weird cryptography failures on Alpine 3.15 that are either going to have me going WAY back to cryptography 3.2 or upgrading pyOpenSSL, and acmebot doesn't like pyOpenSSL :-( ... not a problem I was hoping to run into right at the outset of a large weekend project :'(

**Comment by hyacin75** [comment]
I just did `pip install --upgrade cryptography pyOpenSSL` and then edited the requirements.txt to remove the <20 on pyOpenSSL and it's all running quite happily now
